web/gui/dashboard_info.js

```diff
@@ -3049,44 +3049,83 @@ netdataDashboard.context = {
+    },
+
+    'ebpf.udp_functions': {
+        title : 'UDP calls',
+        info: 'Number of calls for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>, whether they return successfully or fail.'
+    },
+
+    'ebpf.udp_bandwidth': {
+        title : 'UDP bandwidth',
+        info: 'Number of bytes sent and received for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>'
+    },
+
     'ebpf.file_descriptor': {
-        info: 'File descriptor shows the number of calls for internal functions on Linux kernel. The open dimension is attached to the kernel internal function \'do_sys_open\', that is the common function called from open(2) and openat(2). The close dimension is attached to the function \'__close_fd\', that is called from system call close(2).'
+        title : 'File access',
+        info: 'Number of calls for internal functions on Linux kernel. The open dimension is attached to the kernel internal function <code>do_sys_open</code>, which is the common function called from'+
+            ' <a href="https://www.man7.org/linux/man-pages/man2/open.2.html" target="_blank">open(2)</a> ' +
+            ' and <a href="https://www.man7.org/linux/man-pages/man2/openat.2.html" target="_blank">openat(2)</a>. ' +
+            ' The close dimension is attached to the function <code>__close_fd</code>, which is called from system call' +
+            ' <a href="https://www.man7.org/linux/man-pages/man2/close.2.html" target="_blank">close(2)</a>. '
     },
 
     'ebpf.file_error': {
-        info: 'File error shows the number of calls that returned an error when called per period.'
+        title : 'File access error',
+        info: 'Number of calls that returned an error when called per period.'
     },
 
     'ebpf.deleted_objects': {
-        info: 'Deleted objects monitors calls to the function \'vfs_unlink\'. This chart does not show all events to remove files from the file system, because file systems can create their own functions to remove files.'
+        title : 'VFS remove',
+        info: 'Number of calls to the function <code>vfs_unlink</code>. This chart does not show all events that remove files from the file system, because file systems can create their own functions to remove files.'
     },
 
     'ebpf.io': {
-        info: 'IO shows the number of calls for functions \'vfs_read\' and \'vfs_write\' independent of the return to be success or fail. Like the chart \'deleted_objects\', case the file system uses other function to store data on disks, this chart will not show events for it.'
+        title : 'VFS IO',
+        info: 'Number of calls for functions <code>vfs_read</code> and <code>vfs_write</code>, whether they return successfully or fail. As with the chart \'deleted_objects\', this chart will not show events if the file system uses other functions to store data on disk.'
```

Comment:
I think you can remove `number of ` from every description, it is clear from the context. In that specific case 

> whether they return successfully or fail

is not needed to imo, it is clear too.

---

> Number of calls for functions <code>vfs_read</code> and <code>vfs_write</code>, whether they return successfully or fail.

> <code>vfs_read</code> and <code>vfs_write</code> function calls.


Less text is better. @joelhans what do you think?

Reply:
Ok, i reread the description and it seems there is 0 useful info.

> vfs_read and vfs_write function calls.

Is clear from the chart itself.

> As with the chart \'deleted_objects\', this chart will not show events if the file system uses other functions to store data on disk.'

Mentioning of other chart doesnt give you any useful info.

> this chart will not show events if the file system uses other functions to store data on disk.

Well, obvious! If there is no _vfs_read and vfs_write function calls_ the chart show nothing!

Reply:
I will wait for @joelhans here, the other suggestion I already accepted and I changed it.

Reply:
I agree that we could remove `Number of` from every description.

@ilyam8 I disagree that this provides no info. The dimensions for this chart are `read` and `write`, and I'm not convinced all users will immediately understand that they map exactly to `vfs_read` and `vfs_write`.

> Well, obvious! If there is no vfs_read and vfs_write function calls the chart show nothing!

Without the description the user might think there is no filesystem activity at all, which may not be true.

Here's my take on what this should be:

```
Successful or failed calls to functions <code>vfs_read</code> and <code>vfs_write</code>. This chart may not show all file system events if it uses other functions to store data on disk.
```

Discuss!